hypha/apply/activity/templates/messages/email/activity_summary.md

```diff
@@ -0,0 +1,28 @@
+{% extends "messages/email/base.html" %}
+{% load humanize i18n %}
+
+{% block salutation %}{% trans "Dear" %},{% endblock %}
```

Comment:
Wondering if some kind of header makes more sense here, we do not have a specific user to say "Dear nn" too.

What do you think about setting it too `# {% trans "Activities Summary" %}` or similar?

Reply:
That would work; let me know if this sounds better.

---

Activities Summary
--------------------

There are two new activities:


```
Activities Summary
--------------------

There is a new activity:

```

Reply:
![Screenshot 2022-10-14 at 06 12 24@2x](https://user-images.githubusercontent.com/236356/195775499-804a0343-e8c0-489f-8ec7-e095343f89a4.jpg)

Reply:
Updated in https://github.com/HyphaApp/hypha/pull/2991/commits/67cb1a9e80433fe7ba795771846cee34bea7e293

Reply:
Good improvement!